packages/tokens/src/themeToTokensObject.test.ts

```diff
@@ -1,11 +1,21 @@
 import { webLightTheme } from './themes/web/lightTheme';
 import { themeToTokensObject } from './themeToTokensObject';
 import { tokens } from './tokens';
+import { Theme } from './types';
+
+function assertKeys(generatedTokens: Record<keyof Theme, string>, expectedTokens = tokens) {
+  const tokensKeys = Object.keys(expectedTokens) as (keyof Theme)[];
+
+  (Object.keys(generatedTokens) as (keyof Theme)[]).forEach(token => {
+    expect(tokensKeys.includes(token)).toBe(true);
+  });
+}
 
 describe('themeToTokensObject', () => {
   it('passing any of our default themes to the function generates the default tokens object', () => {
     const generatedTokens = themeToTokensObject(webLightTheme);
-    expect(generatedTokens).toEqual(tokens);
+
+    assertKeys(generatedTokens);
```

Comment:
```suggestion
  expect(tokens).toMatchObject(generatedTokens);
```

there is a built-in assertion for this ;)

Reply:
I added another check to test the object format (rather than the full value)